Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -651,6 +650,10 @@
 		67F73E752267B9070079DEEF /* TalkPageReplyListViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 67F73E742267B9070079DEEF /* TalkPageReplyListViewController.swift */; };
 		67F73E792267B9510079DEEF /* TalkPageTopicNewViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 67F73E782267B9500079DEEF /* TalkPageTopicNewViewController.swift */; };
 		67F9AE4923AD2F38003D4F5E /* Array+SafeIndex.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7A3EE1532267DC3800709CF6 /* Array+SafeIndex.swift */; };
+		7016040E2684DE32009C84EC /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7016040D2684DE32009C84EC /* AppDelegate.swift */; };
```

Comment:
Can you also add the new files (`main.swift`, `AppDelegate.swift`, `WMFSettingsViewController.swift`) to our other 3 targets? These are just some alternative builds, they won't compile without these checked:

![Screen Shot 2021-09-17 at 12 06 23 PM](https://user-images.githubusercontent.com/3620196/133827226-eb7b9a7d-e499-40fc-880e-00bd56ce9bd3.png)